examples/experimental/v1beta1/http-redirect-rewrite/httproute-rewritepath.yaml

```diff
@@ -0,0 +1,21 @@
+#$ Used in:
+#$ - site-src/api-types/httproute.md
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: HTTPRoute
+metadata:
+  name: http-filter-rewrite
+spec:
+  hostnames:
+    - rewrite.example
+  rules:
+    - filters:
+      - type: URLRewrite
+        urlRewrite:
+          hostname: elsewhere.example
+	        path:
+            type: ReplacePrefixMatch
+	          replacePrefixMatch: /fennel
```

Comment:
Your indentation looks off here

Reply:
Hi @robscott , Tested and fixed the indentation in the yaml file